invenio_cli/commands/assets.py

```diff
@@ -135,26 +163,40 @@ def link_js_module(self, path):
         steps = [
             FunctionStep(  # Install dependent packages
                 func=self._npm_install_command,
-                args={"path": path},
+                args={"path": path, "module_pkg": module_pkg},
                 message="Installing dependent packages...",
             ),
             FunctionStep(  # Run build script
                 func=self._build_script,
                 args={"module_pkg": module_pkg},
                 message="Building...",
             ),
-            FunctionStep(  # Create link to global folder
-                func=self._run_script,
-                args={"module_pkg": module_pkg},
-                message="Linking module to global dist...",
-            ),
-            FunctionStep(  # Link the global folder to the assets folder.
-                func=self._assets_link,
-                args={"assets_pkg": assets_pkg, "module_pkg": module_pkg},
-                message="Linking module to assets...",
-            ),
         ]
 
+        if self.cli_config.javascript_package_manager.name == "npm":
```

Comment:
would it be better to move that to the JavascriptPackageManager classes? it seems strange to have the classes and do such distinguishes here

Reply:
this entire block is from the WIP commit which is only experimental so far.